docs/UserGuide.md

```diff
@@ -59,15 +59,36 @@ If you can type fast, Tutor's pet can get your class management tasks done faste
 
 </div>
 
-### Viewing help : `help`
+## Command summary
+
+Action | Format, Examples
+--------|------------------
+**Help** | `help`
+**Add Student** | `add-student n/NAME t/TELEGRAM_HANDLE e/EMAIL [tag/TAG]…​` <br> e.g., `add-student n/John Doe t/@johndoe e/johnd@example.com tag/student`
+**List All Students** | `list`
+**Edit Student** | `edit-student INDEX [n/NAME] [t/TELEGRAM_HANDLE] [e/EMAIL] [tag/TAG]…​`<br> e.g., `edit-student 2 n/James Lee e/jameslee@example.com`
+**Find Student** | `find-student KEYWORD [MORE_KEYWORDS]`<br> e.g., `find-student James Jake`
+**Delete Student** | `delete-student INDEX`<br> e.g., `delete-student 3`
+**Clear All Students** | `clear-student`
+**Add Class** | `add-class n/CLASS_NAME`<br> e.g., `add-class n/CS2103T Tutorial T10`
+**List Students In A Class** | `list-students c/INDEX`<br> e.g., `list-students c/3`
+**Edit Class** | `edit-class INDEX n/CLASS_NAME` <br> e.g., `edit 1 n/CS2103T Tutorial T10`
+**Find Class** | `find-class KEYWORD [MORE_KEYWORDS]`<br> e.g., `find-class CS2103T`
+**Delete Class** | `delete-class INDEX`<br> e.g., `delete-class 2`
+**Clear All Classes** |`clear-class`
+**Linking Student To A Class** | ``link s/STUDENT_INDEX c/CLASS_INDEX``<br> e.g., `find Betsy link s/1 c/2`
+**Unlink Student From A Class** | `unlink s/STUDENT_INDEX c/CLASS_INDEX` <br> e.g., `unlink s/1 c/2`
+**Exit** | `exit`
```

Comment:
Hmm, I think summary should remain after the detailed explanation, since it serves so summarise the above mentioned information.

Reply:
I think it'd be better if the summary comes before the detailed explanation. As a user i wouldn't want to scroll downwards if i just want to reference the commands. What do the rest think?

Reply:
Hmm... I think putting it at the top would be more convenient for users to just take a quick look... But the word 'summary' makes it feel like we concluding something so its weird for it to be at the top... Why not change the heading to like 'Command List' or something equivalent?

Reply:
Changed Command summary to Command list